packages/ra-ui-materialui/src/input/DateInput.tsx

```diff
@@ -36,6 +36,11 @@ import { InputHelperText } from './InputHelperText';
  * // The input will display '2021-09-11' whatever the browser timezone.
  *
  * @example
+ * // If you want to manipulate the value from the field to adjust its timezone, use format prop
+ * <DateInput source="published_at" format={value => new Date(value).toISOString().split("T")[0} />
+ * // The input will display the UTC day whatever the browser timezone.
```

Comment:
```suggestion
 * // If you want to manipulate the value from the field to adjust its timezone, use the format prop
 * <DateInput source="published_at" format={value => new Date(value).toISOString().split("T")[0] />
 * // The input will display the UTC day regardless of the browser timezone.
```

Reply:
sure, amended for the other examples too